gkeys/init.lua

```diff
@@ -23,7 +23,23 @@ local hook = win32.SetWindowsHookEx(win32.WH_KEYBOARD_LL,
 			local kbdStruct = ffi.cast("KBDLLHOOKSTRUCT*", lParam)
 
 			local code = kbdStruct.vkCode
-			local name = keymap[code]
+
+			-- Find the keyname from the scancode
+			local bit = require("bit")
```

Comment:
This require should be bumped up to be just after the `jit.os` check, which gives us an earlier error if things go wrong